autoendpoint/src/routers/apns/router.rs

```diff
@@ -575,7 +575,7 @@ mod tests {
     }
 
     /// Create a router for testing, using the given APNS client
-    fn make_router(client: MockApnsClient, db: Box<dyn DbClient>) -> ApnsRouter {
+    async fn make_router(client: MockApnsClient, db: Box<dyn DbClient>) -> ApnsRouter {
```

Comment:
This doesn't need async now with the other PR's changes so this and this entire module's changes can be reverted
```suggestion
    fn make_router(client: MockApnsClient, db: Box<dyn DbClient>) -> ApnsRouter {
```

Reply:
Thanks! missed that one.

Reply:
Hmm... I realized while doing this that `apns::make_router()` is sync, while `fcm::make_router()` is async. I'm tempted to convert the `apns` version to async so that we can make this a trait or at least be consistent.